cns/restserver/internalapi.go

```diff
@@ -225,10 +225,7 @@ func (service *HTTPRestService) ReconcileNCState(
 	if returnCode != types.Success {
 		return returnCode
 	}
-	returnCode = service.UpdateIPAMPoolMonitorInternal(scalar, spec)
-	if returnCode != types.Success {
-		return returnCode
-	}
+	service.IPAMPoolMonitor.Update(scalar, spec)
```

Comment:
unchecked error?

Reply:
`Update` can't return an error, that was the impetus of this simplification